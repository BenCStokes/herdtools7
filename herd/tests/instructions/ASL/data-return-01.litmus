ASL no data return

{}

func f(x::integer) => integer
begin
  return 3;
end

func main()
begin
  let a = f (3);
end

