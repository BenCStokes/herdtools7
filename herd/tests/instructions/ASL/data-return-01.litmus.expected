Test no Required
States 1

Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (true)
Observation no Always 1 0
Hash=699c9c4fff39f51e1815ef4370c4ef47

